your-first-meeting.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: default
+title: Attending your first PR Review Club Meeting
+---
+
+## Attending your first PR Review Club Meeting
+
+If you’re thinking about attending PR Review Club, welcome!  We love new
+participants. Below is information to help you get started as a first timer.
+
+The club meets on {{ site.meeting_day }}s at {{ site.meeting_time }} on the
+`#bitcoin-core-pr-reviews` IRC channel on
+[freenode](https://webchat.freenode.net/). Every week, a Bitcoin protocol
+developer will host a 60 minute discussion on a Bitcoin Core pull request. [The
+code of conduct](https://bitcoincore.reviews/code-of-conduct) details the
+behavior we expect from all participants.
+
+<strong>If you’re new to IRC (Internet Relay Chat),</strong> it has its
+own little culture and etiquette.
+
+The basics:
+
+- Follow the Golden Rule: treat others as you’d like to be treated.
+- Assume the best in others (tone is hard over text)
+- Talk in full sentences, but don't stress too much about grammar or
+  capitalization!
+- To respond directly to someone, just start your message with their full
```

Comment:
perhaps omit "just" and "full"

I'm not sure the sentence about @ them is needed